yaptide/routes/common_sim_routes.py

```diff
@@ -83,6 +77,32 @@ def get(user: UserModel):
         return yaptide_response(message=f"Job state: {job_info['job_state']}", code=200, content=job_info)
 
 
+def get_single_estimator(sim_id: int, estimator_name: str):
+    """Retrieve a single estimator by simulation ID and estimator name"""
+    estimator = fetch_estimator_by_sim_id_and_est_name(sim_id=sim_id, est_name=estimator_name)
+    if not estimator:
+        return yaptide_response(message="Estimator not found", code=404)
+
+    pages = fetch_pages_by_estimator_id(est_id=estimator.id)
+    estimator_dict = {"metadata": estimator.data, "name": estimator.name, "pages": [page.data for page in pages]}
+    return yaptide_response(message=f"Estimator '{estimator_name}' found", code=200, content=estimator_dict)
+
+
+def get_all_estimators(sim_id: int, job_id: str):
+    """Retrieve all estimators for a given simulation ID"""
+    estimators = fetch_estimators_by_sim_id(sim_id=sim_id)
+    if len(estimators) == 0:
+        return yaptide_response(message="Results are unavailable", code=404)
+
+    logging.debug("Returning results from database")
+    result_estimators = []
+    for estimator in estimators:
+        pages = fetch_pages_by_estimator_id(est_id=estimator.id)
+        estimator_dict = {"metadata": estimator.data, "name": estimator.name, "pages": [page.data for page in pages]}
+        result_estimators.append(estimator_dict)
+    return yaptide_response(message=f"Results for job: {job_id}", code=200, content={"estimators": result_estimators})
```

Comment:
Do we really need to pass `job_id` here ? It seems to be used only in the message in the response, where it's not that much needed.